Gemfile.lock

```diff
@@ -15,11 +48,16 @@ GEM
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.7.0)
     rspec-support (3.7.1)
+    ruby_dep (1.5.0)
+    shellany (0.0.1)
+    thor (0.20.0)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
+  guard (~> 2.14)
```

Comment:
Is this dependency necessary for your implementation? If it's not the case, it can be good to make it part of another PR